src/Framework/Runner/AssemblyLoader.cs

```diff
@@ -21,12 +22,19 @@ public AssemblyLoader(RTFAssemblyResolver resolver)
 
         public AssemblyData ReadAssembly(string assemblyPath, GroupingType groupType, string workingDirectory)
         {
-            var assembly = Assembly.ReflectionOnlyLoadFrom(assemblyPath);
+            var assembly = Assembly.LoadFrom(assemblyPath);
```

Comment:
When the loaded Assembly is the RevitSystemTests.dll or RevitNodesTests.dll, the following code assembly.GetTypes() will throw an exception: Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information. I think that's because some references of the two dll files, like SystemTestServices, won't be resolved by RTF, they will be resolved by DynamoRevitDS.dll in D4R.

Reply:
Thank you @ZiyunShang for reviewing!
I was wondering why the `ReflectionOnlyLoadFrom` was used over `LoadFrom` - I understand why now.
I will look into undoing this change and let you know when it's done

Reply:
@ZiyunShang : I have updated the pull request to undo this change. It should now work for your use case. Can you please verify?